crates/query-engine/translation/src/translation/helpers.rs

```diff
@@ -202,6 +203,18 @@ impl<'request> Env<'request> {
             })
     }
 
+    /// Lookup type representation of a type.
+    pub fn lookup_type_representation(
+        &self,
+        scalar_type: &metadata::ScalarType,
+    ) -> Option<metadata::TypeRepresentation> {
+        self.metadata
+            .type_representations
+            .0
+            .get(scalar_type)
+            .cloned()
```

Comment:
Stupid nitpick: we only use this as a reference so we could probably return one and not need to clone it.

This is stupid because cloning it will be cheap.

Reply:
smart. thanks!